vault/dendron.tutorial.md

```diff
@@ -2,28 +2,27 @@
 id: 678c77d9-ef2c-4537-97b5-64556d6337f1
 title: Getting Started
 desc: ''
-updated: 1631317031955
+updated: 1639190763313
 created: 1608051264282
 nav_order: 1.1
 ---
 
-
 ## Pre-requisites
 
 This walkthrough will help you get started with Dendron. Before you start, make sure you do the following:
 - installed [VSCode](https://code.visualstudio.com/download) or a VSCode compatible editor like [VSCodium](https://vscodium.com/)
-- installed [Dendron](https://marketplace.visualstudio.com/items?itemName=dendron.dendron) from the code marketplace
+- installed [Dendron](https://link.dendron.so/vscode) from the code marketplace
 - initialize your first Dendron workspace
 
-When you first install Dendron, you can click on the green `Initialize Workspace` button on the Welcome screen to initialize your first workspace. Then, pick a location to store your notes, and you'll be brought to the tutorial experience, which will contain notes with the same tutorial content here.
+When you first install Dendron, you can click on the green `Get Started` button on the Welcome screen to initialize your first workspace. Then, pick a location to store your notes, and you'll be brought to the tutorial experience, which will contain notes with the same tutorial content here.
 
-![Welcome Screen](https://org-dendron-public-assets.s3.amazonaws.com/images/tutorial-welcome-screen.png)
+![[dendron://dendron.dendron-site/asset.preview#tutorial-welcome-screen,1:#*]]
 
-> 💡 If you don't see this Welcome screen, then you can open it back up by typing `Ctrl+P / Cmd+P` to bring up the command palette, and then running `Dendron: Launch Tutorial` to bring up this Welcome screen again.
+> 💡 If you don't see this Welcome screen, then you can open it back up by typing `Ctrl+Shift+P` / `Cmd+Shift+P` to bring up the [command palette](https://code.visualstudio.com/docs/getstarted/userinterface#_command-palette), and then running `Dendron: Launch Tutorial` to create a workspace.
 >
-> ![Welcome Screen](https://org-dendron-public-assets.s3.amazonaws.com/images/tutorial-launch-tutorial-cmd.png)
+> ![[dendron://dendron.dendron-site/asset.preview#tutorial-launch-command,1:#*]]
 
```

Comment:
add a line about "Backlinks" and "Children" since they show up at the bottom. the line could be "we'll get into it later"